packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.2.0] July 12, 2022
```

Comment:
I guess the following PR needs to go first ?
https://github.com/Prime-Holding/rx_bloc/pull/277